prescriptions/co_/connexion/werkzeug_1x0_2x0.yaml

```diff
@@ -0,0 +1,43 @@
+units:
+  steps:
+  - name: Connexion2x10x0Werkzeug1x0Step
+    type: step.Group
+    should_include:
+      adviser_pipeline: true
+    match:
+    - group:
+      - package_version:
+          name: connexion
+          version: ">=2.10.0"
+          index_url: https://pypi.org/simple
+      - package_version:
+          name: werkzeug
+          version: <1.0,>=2.0
+          index_url: https://pypi.org/simple
+    run:
+      stack_info:
+      - &stack_info
+        type: WARNING
+        message: Cannot install connexion >=2.10.0 when using werkzeug <1.0,>=2.0
```

Comment:
If I understand this correctly, we are stating at that **any version above 2.10.0** in connexion can't be installed with werkzeug **<1.0 ,>=2.0**. That is already present in setup.py of connexion with help of: https://github.com/spec-first/connexion/blob/fc6803bc0021c8b9840558d9429bb4c272689ea4/setup.py#L29
till I understand if anyone installing from source, would never fall in this issue.

What according to me would be helpful for our users is stating 
any version above 2.4.0 in connexion should not be installed with werkzeug below 0.14 
as in 2.4.0 connexion, requires_install is not stated with the condition: https://github.com/spec-first/connexion/blob/c8d8973c7e390a5e2071c7b494dd7bd52bd55ff1/setup.py#L23

As, we have found out: 
connexion 2.4.0 is using https://github.com/spec-first/connexion/blob/c8d8973c7e390a5e2071c7b494dd7bd52bd55ff1/connexion/decorators/validation.py#L11
which is introduced in werkzeug 0.15.0: https://github.com/pallets/werkzeug/tree/0.15.0

I maybe wrong, but my understanding of prescription was to have knowledge that was normally missed. 
open for further discussion.

Reply:
Thanks! That is right. I corrected the versions of the packages in https://github.com/thoth-station/prescriptions/pull/20561/commits/3c3ed4fd5d9742e18c0c9c3d5547c5ef97d1fb53 👍🏻